cityofaustin/atd-moped
# Upgrade to react-router v6.3

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/9536

## Summary of changes
1. Migrate to lockfile v2 ([discussion](https://austininnovation.slack.com/archives/CHZE6BC6L/p1655911113444509))
2. Tweak routes to make them compatible with API changes
3. Implement a `<NavLink/>` wrapper component to preserve the `activeClassName` prop
4. Remove side navigation components which we made hidden some time ago.
5. Forcibly bump `react-map-gl-draw`'s peer dependency of React from v16 to v17 (`package-lock.json`)
6. Remove react-hook-form dev tools, which had package conflicts


## Testing
**URL to test:** Netlify and local

**Steps to test:**
We're testing to make sure all of the navigation links work normally. 
1. Test the top nav links: dashboard, projects, staff
6. Click on a project from the project list—make sure all project tabs load
7. Sign out from the drop-down nav

---
#### Ship list
